app/controllers/posts_controller.rb

```diff
@@ -28,6 +28,18 @@ def create
     end
   end
 
+  def destroy
```

Comment:
- [ ] It will be nice to add `  load_and_authorize_resource` at the top of the post_controller.  `load_and_authorize_resource `sets up a `before_filter`  which calls two methods: `load_resource` and `authorize_resource`.